smack-experimental/src/main/java/org/jivesoftware/smackx/httpfileupload/AbstractHttpUploadException.java

```diff
@@ -0,0 +1,101 @@
+/**
+ *
+ * Copyright Micha Kurvers
```

Comment:
Please add the year to the copyright statement. See also the other copyright statements in Smack.